packages/charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -0,0 +1,141 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IVerticalBarChart {
```

Comment:
>IVerticalBarChart [](start = 17, length = 17)

Empty interface?

Reply:
Oh I see this is sort of a pattern we have at least in some places atm, although it is a little weird to see it.

Reply:
It provides a place to add a public interface. I am ok with it, moreso than people not realizing there should be one.